ui/dashboards/src/components/Panel/PanelHeader.tsx

```diff
@@ -77,7 +74,15 @@ export function PanelHeader({ id, title, description, editHandlers, isHovered, s
       aria-describedby={descriptionTooltipId}
       disableTypography
       title={
-        <Typography id={titleElementId} variant="subtitle1">
+        <Typography
+          id={titleElementId}
+          variant="subtitle1"
+          sx={{
+            whiteSpace: 'nowrap',
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+          }}
+        >
```

Comment:
This ensures that the panel name becomes "ellipsified" when it's cut off (when the panel is resized to be small)